src/libjasper/include/jasper/jas_seq.h

```diff
@@ -185,6 +185,12 @@ static inline jas_matind_t jas_matrix_size(const jas_matrix_t *matrix)
 	return jas_matrix_numcols(matrix) * jas_matrix_numrows(matrix);
 }
 
+JAS_ATTRIBUTE_PURE
+static inline bool jas_matrix_empty(const jas_matrix_t *matrix)
```

Comment:
I don't see this anywhere being used.

Reply:
This function is for completeness, because I added the same function for `jas_seq_t`. There are lots of (matrix) functions which are never used.